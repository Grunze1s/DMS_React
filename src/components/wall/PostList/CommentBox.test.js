import React from 'react';
import CommentBox from './CommentBox';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < CommentBox {...props}  />)
    )
}

describe('<CommentBox />', () => {
  

it("CommentBox render", () => { 
        const wrapper= setup();
        expect(wrapper).toMatchSnapshot();
  
})
})

