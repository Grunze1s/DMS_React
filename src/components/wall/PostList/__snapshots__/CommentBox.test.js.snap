// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentBox /> CommentBox render 1`] = `ShallowWrapper {}`;
